.pre-commit-config.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2022 Obscurity Labs LLC. <admin@obscuritylabs.com> - All Rights Reserved
+# Copyright (C) 2023 Obscurity Labs LLC. <admin@obscuritylabs.com> - All Rights Reserved
```

Comment:
I don't think we can copyright these files or all the others of this type (not ours).